2.adoc

```diff
@@ -72,5 +72,165 @@ Optional description
+  ** klávesové skratky na styling dokumentu (nadpisy, odsadenie, tabulky, bloky kódu, citacie, zoznamy, foot notes, horizontal lines, bold, underline,..., vloženie obrázku) (neaktívne v source code mode)
+* Nepodporuje režim so zobrazeným zdrojákom a vyrenderovaným dokumentom vedľa seba
+  ** štandardne live rendering v tom istom okne (preview)
+  ** source code mod pridáva nadštandardný syntax highlighting zdrojaku (niečo ako vyrenderovaný dokument akurát je v ňom stále markdown)
+* Chýba statusbar
+  ** Chýba informácia o pozícií kurzora v dokumente
+  ** Auto-save na macOS v réžií systému, bez informácie o poslednom uložení
+* Outline 
+  ** chýba presúvanie kapitol potiahnutím nadpisu v outline
+* Práca sa tabuľkami
+  ** UI dialog pre vloženie tabuľky (počet riadkov, sĺpcov)
+  ** Tabuľka je hneď vyrenderovaná a do buniek je možné vpisovať hodnoty
+  ** Malý toolbar pre výber zarovnania textu v bunkách tabuľky a pohodlný resize
+
+image::https://i.imgur.com/1uSz1xM.png[]
+
+* Vkladanie obrázkov
+  ** cez menu (otvorí sa nativny dialog), klávesovou skratkou (snippet) alebo drag&drop
+  ** možnosť automatického uploadu obrázku na server (nutné nainštalovať ďalšiu aplikáciu)
+
+#### Zajímavé features
+
+(viac menej asi popísané vyššie)
+
+* Pekný syntax highlighting kódu
+* Generovanie Table of Contents v dokumente
+* Themes
+  ** Na výber z niekoľkých tém, ktoré menia vzhľad ako aj vyrenderovaného dokumentu tak aj zdrojového kódu
+* Export do PDF, docx, OpenOffice, LaTeX, MediaWiki, Epub
+
+==== Shrnutí
+
+===== Plusy
+* Príjemný vzhľad, možnosť zmeny vzhľadu
+* live rendering šetrí miesto
+* prijemná práca s tabuľkami
+* vkladanie obrázkov drag&drop
+* podpora tabov
+
+===== Mínusy
+* Málo možnosti konfigurácie
+* Spellchecker nepodtrháva nesprávne slovo
+* Ak užívateĽ nepozná klávesové skratky je nutné siahnuť po myši (chýba command palleta)
+* komerčný editor
+* Nemožnosť nastaviť otvárenie nového tabu namiesto okna
+
+
+=== AsciiDocLIVE
+
+* https://asciidoclive.com
+* online open-source asciiDoc-only editor
+
+==== Porovnání rozdílů
+
+Jedná sa o webovú komponentu, ktorá ale nie je embedovateľná. Podporuje iba jazyk AsciiDoc. Kvôli chýbajúcej command pallete vyžaduje používanie myši ak si užívateľ nepamätá potrebnú klávesovú skratku. Podporuje zmenu vzhľadu pomocou css šablón. Dokument je možné uložiť na Dropbox, Google Drive alebo na filesystém. 
+
+==== Zhodnocení UI
+
+image::https://i.imgur.com/O8bixCn.png[]
+
+* Okno editoru je rozdelené na dve časti
+  ** zdrojový kód so zýraznenou syntaxou (len farebné rozlíšenie)
+  ** vyrenderovaný dokument (live rendering)
+  ** možnosť zapnúť synchronizované scrollovanie v nastaveniach
+
+
+* Klavesové skratky
+  ** možnosť vybrať z 3 presetov (default, vim, emacs)
+  ** UX fail: užívateľ nikde nezistí aké konkrétne klavesové skratky je možné používať 
+
+
+==== Pozorování UI v kontextu markup-editor
+
+* má Toolbar(menu) v hornej časti
```

Comment:
My nemame toolbar ale statusbar, takze to neni podle me konstratst.

![](http://www.charlescooke.me.uk/05-on/oh-eight/images/oh-eight-fig3-1.png)

Reply:
Nerozumiem. V tej sekcii sa mám vyjadriť len k UI prvkom, ktoré mame aj my? Chcel som práve poukázať na to že oni ho majú, my nie.

Reply:
jo takhle, tim padem v poradku. Ja jsem to pochopil tak, ze upozornujes na to umisteni (ktery v pripade status baru maji stejny). Sry.